Sample/GetV2ProviderStatus.ps1

```diff
@@ -0,0 +1,17 @@
+﻿function Get-AmsV2ProviderStatus([string]$subscriptionId, [string]$resourceGroup, [string]$monitorName, [string]$providerName, $logger)
+{
+    $rawToken = Get-AzAccessToken -ResourceTypeName Arm
+    $armToken = $rawToken.Token
+
+    $headers = @{
+        "Content-Type" = "application/json"
```

Comment:
At a later point (we can do this from the next PR ) consider creating constants for common string like "application/json", api urls etc.

Reply:
sure, noted